importpub/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM public.ecr.aws/lambda/python:3.8
+
+RUN pip3 install scholarly
+
+# Copy function code
+COPY handler.py ./
+COPY gscholar.py ./
+
+RUN cd /var/lang/lib/python3.8/site-packages/scholarly && \
+    sed -i '370s/()/(use_cache_server = False, verify_ssl=False)/' _proxy_generator.py
```

Comment:
Does this correctly edit the library contents or did you end up going with editing the file in the running container and committing it as a new image?

Reply:
This does edit the library contents without me having to go into it to change it and make a new image